### PR TITLE
chore: Make graph visible in GUI, and show a proper error message

### PR DESCRIPTION
**Pull request by jorisros**
Related to issue #65

| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | yes/no
| Fixed tickets | #65

### CODE REVIEW

File: src/WorkflowGui/Controller/WorkflowController.php [hunk 1]
```diff
@@ -339,9 +339,14 @@ private function getVisualization($workflow, $format): string
             throw new \InvalidArgumentException($this->trans('workflow_cmd_not_found', ['dot']));
         }
 
+        $workflowRepository = $this->repository->find($workflow);
+
+        if ($workflowRepository && !$workflowRepository['enabled']) {
```
Comment (kjkooistra-youwe) [inline src/WorkflowGui/Controller/WorkflowController.php]: Possibly rewrite as `!($workflowRepository['enabled'] ?? false)`

File: src/WorkflowGui/Resources/translations/admin.en.yml [hunk 2]
```diff
@@ -114,3 +114,4 @@ workflow_gui: 'Workflow GUI'
 workflow_problem_creating_workflow_invalid_characters: 'Please use only these characters: "a-z", "A-Z" and "_"'
 workflow_gui_workflow_with_name_already_exists: 'Workflow with bane already exists'
 workflow_gui_clone: 'Clone'
+workflow_gui_enable_message: 'Please enable and save the workflow, before you can see the graph'
```
Comment (kjkooistra-youwe) [inline src/WorkflowGui/Resources/translations/admin.en.yml]: This sentence is structurally incorrect. Suggestion:
Please enable and save the workflow to display graph visualization.

or something similar.

Comment (jorisros) [reply src/WorkflowGui/Resources/translations/admin.en.yml]: Ah my bad English again, thank you

File: src/WorkflowGui/Controller/WorkflowController.php (hunk 1 above)
Comment (jorisros) [reply src/WorkflowGui/Controller/WorkflowController.php]: I did split it up in two messengers, so it become more clear for enduser

File: src/WorkflowGui/Resources/translations/admin.en.yml (hunk 2 above)
Comment (kjkooistra-youwe) [reply src/WorkflowGui/Resources/translations/admin.en.yml]: I don't see an update yet.

Comment (jorisros) [reply src/WorkflowGui/Resources/translations/admin.en.yml]: @kjkooistra-youwe should be oke now

**Review by kjkooistra-youwe — APPROVED** [review]

**Pull request merged**